src/QMCHamiltonians/ObservableHelper.h

```diff
@@ -35,10 +35,11 @@ using value_type = QMCTraits::FullPrecRealType;
 class ObservableHelper
 {
 public:
+  ObservableHelper() = delete;
```

Comment:
This is not really necessary. Once there is an explicit constructor. The one with empty argument should be disabled by default.

Reply:
It's true I put this in to make sure there wasn't a hidden use of the default constructor somewhere.  I will remove it.